README.md

```diff
@@ -36,6 +36,7 @@ Here's the current selection of Quarkus Azure Services:
 Besides extensions for specific Azure services, there are some common extensions that are used by the Azure services extensions or can be used independently as well:
 
 * [Azure Identity Extension](common/azure-identity): This extension provides a way to authenticate with Azure services using the [azure-identity](https://mvnrepository.com/artifact/com.azure/azure-identity) library.
+* [Azure Identity JDBC Extension](common/azure-identity-jdbc): This extension provides a way to authenticate Azure Database for PostgreSQL/MySQL Flexible server with Microsoft Entra authentication using the [azure-identity-extensions](https://mvnrepository.com/artifact/com.azure/azure-identity-extensions) library.
```

Comment:
Perhaps this reference is better:
```suggestion
* [Azure Identity JDBC Extension](common/azure-identity-jdbc): This extension provides a way to authenticate Azure Database for PostgreSQL/MySQL Flexible server with Microsoft Entra authentication using the [Azure identity authentication extensions plugin library for Java](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/identity/azure-identity-extensions).
```

Reply:
Yeah, it's much better. Thank you @albers.